json_examples/task-min_eeg.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It would be better if this file lives in the same folder as other fMRI/MEG templates:
BIDS-Starter-Kit/templates/sub-01/ses-01/eeg/sub-01_ses-01_task-ShortExample_run-01_eeg.json

These templates are meant to be empty.

Reply:
+1, moving the files [here](https://github.com/INCF/BIDS-Starter-Kit/tree/master/templates/sub-01/ses-01) and creating a new `/eeg` directory there. We'll then be able to add more EEG templates over there